web/samples/timeout.js.php

```diff
@@ -0,0 +1,3 @@
+<?php 
```

Comment:
We can replace the .php with a .js that simulates the timeout: it cancels the `load` event.

````
(function() {
  var me = document.currentScript;
  document.body.addEventListener('load', function(e) { 
    // Prevent the element's onload event from firing
    if(e.srcElement == me) {
      e.cancelBubble = true;
    }
  }, {capture: true});
})();
````